server/src/main/kotlin/com/github/neho4y/category/service/impl/CategoryServiceImpl.kt

```diff
@@ -0,0 +1,48 @@
+package com.github.neho4y.category.service.impl
+
+import com.github.neho4y.category.domain.Category
+import com.github.neho4y.category.domain.repository.CategoryRepository
+import com.github.neho4y.category.model.CategoryCreationDto
+import com.github.neho4y.category.model.CategoryDto
+import com.github.neho4y.category.service.CategoryService
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.lang.RuntimeException
+
+
+@Service
+class CategoryServiceImpl(
+        private val categoryRepository: CategoryRepository) : CategoryService {
```

Comment:
Check class constructor formatting here. If it is multiline, then closing bracket should be on a separate line.